python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -829,13 +836,37 @@ async def arun(self, checkpoint: bool = False) -> None:
                 self.offset_total_coma_y += coma_y
                 await self.atcs.offset_aos_lut(z=focus_offset, x=coma_x, y=coma_y)
 
+            # Not sure if here is the best place to get number of executed
+            # iterations
+            self.executed_iterations += 1
+
+        # If we reach this point, it means we did not converge.
         self.log.warning(
-            f"Reached maximum iteration ({self.max_iter}) without convergence.\n"
+            f"Reached maximum iteration ({self.max_iter}) without convergence."
+            f"Returning hexapod to initial position. \n"
+        )
+
+        self.rem.ataos.evt_detailedState.flush()
+
+        # Return hexapod to original position
+        await self.atcs.rem.ataos.cmd_offset.set_start(
```

Comment:
I am not sure this is going to do what you expect. The `cmd_offset` command is the same command used in `atcs.offset_aos_lut` and what it does is to apply offsets to the hexapod, not set the position. 

So I think this might be missing some steps where it gets the current position of the hexapod and undoes the offsets.

Also, I see you "reset" the hexapod position in 2 different locations, maybe extract this into a `reset_hexapod` method? I would recommend using the `atcs.offset_aos_lut` method though, instead of the command directly.

Reply:
I got it. There are already variables called `offset_total_focus/coma_x/coma_y` that could be used to return the hexapod back to its original position by applying the total offsets in the reverse direction, like this:

```
await self.atcs.offset_aos_lut(z=-offset_total_focus, x=-offset_total_coma_x, y=-offset_total_coma_y)
```

Does it make sense for you?

Reply:
I implemented that using the `offset_total_focus/coma_x/coma_y` variables as entry to the `atcs.offset_aos_lut`. This may return the hexapod to the initial position, removing the total computed offsets.